docs/the_guard.md

```diff
@@ -1,6 +1,6 @@
 # What is a Guard?
 
-The guard object is the main interface for GuardRails. It is seeded with a RailSpec, and then used to run the GuardRails AI engine. It is the object that accepts changing prompts, wraps LLM prompts, and keeps track of call history.
+The guard object is the main interface for GuardRails. It can be used without configuration for string-based LLM apps, and accepts a pydantic object for structured data usecases. The Guard is then used to run the GuardRails AI engine. It is the object that wraps LLM calls, orchestrates validation, and keeps track of call history.
```

Comment:
should we update this to also describe how we can do input message/prompt/instruction validation too?

Reply:
nvm i can see it through the parse flow